apps/mosaico/src/Ads/Ad.purs

```diff
@@ -27,24 +34,34 @@ type Props =
 
 type State =
   { populated :: Boolean
+  , gamId :: Maybe String
+  , isLazy :: Maybe Boolean
   }
 
 ad :: Props -> JSX
 ad = make component
-  { initialState: { populated: false }
+  { initialState: { populated: false, gamId: Nothing, isLazy: Nothing }
   , didMount: \self -> do
       runEffectFn1 fetchAdImpl self.props.contentUnit
-
-  , render: \self ->
-      DOM.div
-        { className: blockClass <> " " <> toLowerCase self.props.contentUnit
-        , children:
-          [ DOM.div
-              { id: self.props.contentUnit
-              , className: blockClass <> "__content-unit"
-              }
-          ]
-        }
-      }
-      where
-        blockClass = "mosaico-ad"
+      gamId <- runEffectFn1 getGamId self.props.contentUnit
+      isLazy <- runEffectFn1 getIsLazy self.props.contentUnit
+      self.setState _ { gamId = toMaybe gamId, isLazy = toMaybe isLazy }
+  , render: render
+  }
+    where
+      blockClass = "mosaico-ad"
+      networkCode = "21664538223"
+      render { state: { gamId: Nothing, isLazy: Nothing }} = mempty
```

Comment:
We can avoid this repetition. Remove these and instead add `render _ = mempty` after the `Just ... Just ...` case.